jax/_src/numpy/lax_numpy.py

```diff
@@ -986,7 +986,10 @@ def heaviside(x1, x2):
 def hypot(x1, x2):
   _check_arraylike("hypot", x1, x2)
   x1, x2 = _promote_dtypes_inexact(x1, x2)
-  return lax.sqrt(x1*x1 + x2*x2)
+  x1 = lax.abs(x1)
+  x2 = lax.abs(x2)
+  x1, x2 = where(x1 > x2, x1, x2), where(x1 < x2, x1, x2)
```

Comment:
Can we make this:
```
x1, x2 = lax.max(x1, x2), lax.min(x1, x2)
```
?

Reply:
I tried that, but tests failed. I don't remember the exact reason, but I believe it had something to do with max and min not being broadcast-able.

Reply:
Ah. What about:
```
x1, x2 = maximum(x1, x2), mininimum(x1, x2)
```

?

That uses the `maximum` and `minimum` defined in this file, and they should both broadcast.

Reply:
Done, tests pass locally now!